### PR TITLE
Add APIs: storagefee getnetworkfee, tickets tools gettotalstoragefee

### PR DESCRIPTION
**Pull request by vadymeng0**
This commit is to implement: 
    + https://freedcamp.com/view/2956206/tasks/40506963
    + https://freedcamp.com/view/2956206/tasks/40507034
Follow the code of `storagefee getnetworkfee` and add similar logic for `storagefee getartticketfee`
  - Create method CAmount CMasterNodeController::GetArtTicketFeePerKB()
  - Use `ArtTicketFeePerKBDefault` for default value and set it to 3
Change MasternodeFeePerMBDefault to 50

### CODE REVIEW

File: src/mnode/mnode-rpc.cpp [hunk 1]
```diff
@@ -2241,6 +2250,97 @@ As json rpc
...
+		"mn2":{"mn2PastelID":"mn2Signature"},
+		"mn2":{"mn3PastelID":"mn3Signature"}
+	}
+3. "pastelid"   (string, required) The current, registering masternode (MN1) PastelID. NOTE: PastelID must be generated and stored inside node. See "pastelid newkey".
+4. "passpharse" (string, required) The passphrase to the private key associated with PastelID and stored inside node. See "pastelid newkey".
+5. "key1"       (string, required) The first key to search ticket.
+6. "key2"       (string, required) The second key to search ticket.
+7. "fee"        (int, required) The agreed upon storage fee.
+8. "imagesize"  (int) size of image in MB
+Masternode PastelID Ticket:
```
Comment (a-ok123) [inline src/mnode/mnode-rpc.cpp:2282]: This part is not needed here - lines 2282-2299

File: src/mnode/mnode-rpc.cpp [hunk 2]
```diff
@@ -2241,6 +2250,97 @@ As json rpc
...
+	"txid": ""
+}
+
+Register Art Ticket
+)" + HelpExampleCli("tickets tools gettotalstoragefee", R"(""ticket-blob" "{signatures}" jXYqZNPj21RVnwxnEJ654wEdzi7GZTZ5LAdiotBmPrF7pDMkpX1JegDMQZX55WZLkvy9fxNpZcbBJuE8QYUqBF "passphrase", "key1", "key2", 100, 3)") +
+                                           R"(
+As json rpc
+)" + HelpExampleRpc("tickets", R"("tools", "gettotalstoragefee", "ticket" "{signatures}" "jXYqZNPj21RVnwxnEJ654wEdzi7GZTZ5LAdiotBmPrF7pDMkpX1JegDMQZX55WZLkvy9fxNpZcbBJuE8QYUqBF" "passphrase", "key1", "key2", 100, 3)"));
+
+                if (!masterNodeCtrl.IsActiveMasterNode())
```
Comment (a-ok123) [inline src/mnode/mnode-rpc.cpp:2307]: Sorry, i didn't clear it out before - anyone can make this call

**Review by a-ok123 — CHANGES_REQUESTED** [review]

File: src/mnode/mnode-rpc.cpp (hunk 1 above)
Comment (vadymeng0) [reply src/mnode/mnode-rpc.cpp:2282]: I removed it

File: src/mnode/mnode-rpc.cpp (hunk 2 above)
Comment (vadymeng0) [reply src/mnode/mnode-rpc.cpp]: I removed it

**Review by a-ok123 — APPROVED** [review]

**Pull request merged**